frontend/src/components/custom-tools/view-tools/ViewTools.jsx

```diff
@@ -89,4 +93,16 @@ ViewTools.propTypes = {
   type: PropTypes.string,
 };
 
+ViewTools.defaultProps = {
```

Comment:
@tahierhussain It is recommended to use default values instead or defaultProps for functional componente